examples/netlify-functions/functions/removeFromCart/removeFromCart.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const { Cart } = require('../../models');
+
+const handler = async(event) => {
+  try {
+    const cart = await Cart.findOne({ _id: event.body.cartId });
+    const index = cart.products.findIndex((item) => {
+      return item._id == event.body.productId;
+    });
+    cart.products.splice(index, 1);
```

Comment:
Don't assume that the product will be found! If `index === -1`, this will have unexpected behavior.

Also, we need to support removing a certain quantity from the cart. Say you have 2 iPhones in your cart. You should be able to just remove 1 iPhone, not both iPhones.